runner.py

```diff
@@ -275,6 +287,25 @@ def process_data(with_indels, dust, evalue, fastq_files, log, max_basecall_itera
     return reference_file
 
 
+def validate_input(output_dir, input_dir, reference_file):
```

Comment:
this should be a separate pr as it is a different and independent feature

Reply:
removed and will be submited as another PR